documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -37,7 +37,13 @@ spec:
 |Label |Description |Default value
 
 |`controller_max_vm_inflight`
-|The maximum number of VMs per plan that can be migrated simultaneously.
+a|*Varies with provider as follows*:
+
+* {rhv-full}, {osp}, VMware warm migrations _or_ remote cold migrations: The maximum number of disks that {project-short} can transfer simultaneously.
```

Comment:
For **all** occurrences of  **remote cold migrations**:
I suggest to explain what does "remote migration" mean  and therefore rephrase to something like:
 **cold migrations to a remote OpenShift cluster**